README.md

```diff
@@ -276,6 +297,80 @@ Custom metrics are also provided for the individual service endpoints and follow
 "timers.<metric-name>.responseTimeHist.p99": 0,
 ```
 
+#### Prometheus metrics
+
+Optimizely Agent also supports Prometheus metrics. Prometheus is an open-source systems for monitoring and alerting toolkit. You can use it to collect and visualize metrics in a time-series database.
+
+To access the Prometheus metrics, you can use the `/metrics` endpoint with a Prometheus server. The metrics are exposed in a format that Prometheus can scrape and aggregate.
+
+Example Request:
+
+```bash
+curl localhost:8088/metrics
+```
+
+This will return a plain text response in the Prometheus Exposition Formats, which includes all the metrics that Prometheus is currently tracking.
```

Comment:
```suggestion
This will return a plain text response in the Prometheus Exposition Format, which includes all the metrics that Prometheus is currently tracking.
```